src/Server/SSH/SSHChannel.cpp

```diff
@@ -80,6 +85,7 @@ bool SSHChannel::isOpen()
 
 void SSHChannel::deleter(ssh_channel ch)
 {
+    UNUSED(ch);
```

Comment:
Why `UNUSED`? `ssh_channel_free` is not a macro just a regular function, so it definitely in use